app/models/cloud_network.rb

```diff
@@ -55,6 +55,57 @@ def self.tenant_id_clause_format(tenant_ids)
     ["((tenants.id IN (?) OR cloud_networks.shared IS TRUE OR cloud_networks.external_facing IS TRUE) AND ext_management_systems.tenant_mapping_enabled IS TRUE) OR ext_management_systems.tenant_mapping_enabled IS FALSE OR ext_management_systems.tenant_mapping_enabled IS NULL", tenant_ids]
   end
 
+  def update_cloud_network_queue(userid, options = {})
+    $nsxt_log.info("update_cloud_network_queue")
```

Comment:
Did you define the new `$nsxt_log` in https://github.com/ManageIQ/manageiq/blob/master/lib/vmdb/loggers.rb ?

Reply:
Also in core we shouldn't be using a plugin specific logger, another network provider might implement this in the future

Reply:
The NSXT logger and other parts will come in a later PR. Is removed.